airbyte-webapp/src/components/JobItem/components/MainInfo.tsx

```diff
@@ -135,10 +140,13 @@ const MainInfo: React.FC<IProps> = ({
           {jobStatus}
           {shortInfo ? <FormattedMessage id="sources.additionLogs" /> : null}
           {attempts.length && !shortInfo ? (
-            <AttemptDetails
-              attempt={attempts[attempts.length - 1]}
-              configType={job.configType}
-            />
+            <div>
+              {attempts.length > 1 && <Text>Last Attempt:</Text>}
```

Comment:
This string still needs to be internationalized.

Reply:
Yeah will do waiting for UI Look to be finalised. What do you think this is fine?

Reply:
Could you maybe attach a screenshot to the PR? It would help already getting a feeling and commenting on design (while waiting for everything to build locally :D)

Reply:
Yep will do